docs/workflows/SNVPhyl/0.1/repository_dependencies.xml

```diff
@@ -8,5 +8,4 @@
 	<repository name="sam_to_bam" owner="devteam" toolshed="http://toolshed.g2.bx.psu.edu/" changeset_revision="c73bf16b45df" />
 	<repository name="samtools_mpileup" owner="devteam" toolshed="http://toolshed.g2.bx.psu.edu/" changeset_revision="973fea5b4bdf" />
 	<repository name="bcftools_view" owner="gregory-minevich" toolshed="http://toolshed.g2.bx.psu.edu/" changeset_revision="6572c40a8505" />
-	<repository name="msa_datatypes" owner="iuc" toolshed="http://toolshed.g2.bx.psu.edu/" changeset_revision="70227007b991" />
```

Comment:
You should really only be modifying this file in the `SNVPhyl/1.1` directory. The sole purpose of this file was to install tools for the virtual machine anyways. I think for the latest SNVPhyl versions it just points to the `suite_snvphyl` tool suite.

Reply:
This change was an accident! Reverting